connector_importer_product/components/product_product/record_handler.py

```diff
@@ -33,6 +33,16 @@ def odoo_post_create(self, odoo_record, values, orig_values):
     def odoo_post_write(self, odoo_record, values, orig_values):
         self._update_template_attributes(odoo_record, values, orig_values)
 
+    def odoo_pre_write(self, odoo_record, values, orig_values):
+        if (
+            "product_tmpl_id" not in orig_values
+            and "product_tmpl_id" in odoo_record._fields.keys()
```

Comment:
```suggestion
            and "product_tmpl_id" in odoo_record._fields
```
and please report here as a comment the explanation of the commit.